frontend/test/named-pulls.ts

```diff
@@ -152,3 +152,20 @@ export const ManyStatuses = <PullData[]> [
       },
    }),
 ];
+
+const devBlock = sig(SignatureType.dev_block, true);
+const deployBlock = sig(SignatureType.deploy_block, true);
+export const Blocked = <PullData[]> [
```

Comment:
Maybe a pull with dev and deploy blocked so we make sure it's in the dev_blocked column?

Reply:
These are (at the moment) only used on the pull-card-demo page. I could probably find a way to use this as the main data-source for the dummy-pulls too, but that can happen in another pull.